install-dev/data/db_structure.sql

```diff
@@ -1647,6 +1647,7 @@ CREATE TABLE `PREFIX_product_attribute_shop` (
   `low_stock_alert` TINYINT(1) NOT NULL DEFAULT 0,
   `available_date` date DEFAULT NULL,
   PRIMARY KEY (`id_product_attribute`, `id_shop`),
+  INDEX (`default_on`),
```

Comment:
In order to fit the upgrade file, can you please give a name to the index?

```sql
INDEX `default_on` (`default_on`),
```

Reply:
Oops totally forgot about the upgrade, will do both ASAP (there is another PR for 1.6 too)